conf/config-default.yaml

```diff
@@ -271,54 +271,65 @@ etcd:
 graphql:
   max_size: 1048576               # the maximum size limitation of graphql in bytes, default 1MiB
 
-plugins:                          # plugin list (sorted in alphabetical order)
-  - api-breaker
-  - authz-keycloak
-  - basic-auth
-  - batch-requests
-  - consumer-restriction
-  - cors
-  # - dubbo-proxy
-  - echo
-  # - error-log-logger
-  # - example-plugin
-  - fault-injection
-  - grpc-transcode
-  - hmac-auth
-  - http-logger
-  - ip-restriction
-  - jwt-auth
-  - kafka-logger
-  - key-auth
-  - ldap-auth
-  - limit-conn
-  - limit-count
-  - limit-req
-  # - log-rotate
-  # - node-status
-  - openid-connect
-  - prometheus
-  - proxy-cache
-  - proxy-mirror
-  - proxy-rewrite
-  - redirect
-  - referer-restriction
-  - request-id
-  - request-validation
-  - response-rewrite
-  - serverless-post-function
-  - serverless-pre-function
-  # - skywalking
-  - sls-logger
-  - syslog
-  - tcp-logger
-  - udp-logger
-  - uri-blocker
-  - wolf-rbac
-  - zipkin
-  - server-info
-  - traffic-split
+#ext-plugin:
+  #cmd: ["ls", "-l"]
 
+plugins:                          # plugin list (sorted by priority)
+  - real-ip                        # priority: 23000
```

Comment:
We can submit a new PR for `priority: 23000... ...`

One PR does only one thing

Reply:
I just wanted to add the declaration of 
```
- ldap-auth                      # priority: 2540
```
i'll update the file with the upstream one and just add this declaration